kfp/kfp_support_lib/src/kfp_support/workflow_support/utils/workflow_utils.py

```diff
@@ -142,6 +142,42 @@ def __init__(self, host: str = "http://localhost:8080"):
         :param host: host to connect to
         """
         self.kfp_client = Client(host=host)
+    
+    def upload_pipeline(
+        self,
+        pipeline_package_path: str = None,
```

Comment:
why it is package? should it be a pipeline yaml file?

Reply:
it is taken from the kfp function https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/client/client.py#L1388